.flake8.rej

```diff
@@ -0,0 +1,9 @@
+diff a/.flake8 b/.flake8	(rejected hunks)
```

Comment:
What are these files?!

Reply:
This is what happens when cruft can't merge the changes due to them not matching the template. It opens the PR with them like this so that we can see the output of cruft running but they require manual fixing.

Reply:
So in general we should keep these open as placeholders to fix it when it fails to do it automatically.